app/src/main/java/it/quip/android/actvitiy/LoginActivity.java

```diff
@@ -49,7 +49,7 @@ public boolean onOptionsItemSelected(MenuItem item) {
         int id = item.getItemId();
 
         //noinspection SimplifiableIfStatement
```

Comment:
Oh its for the IDE. It wants to simplify the if statement if you get rid of that guy